File: Creacion API/APIRest2_23_24_Capas/APIRest2_23_24/src/main/java/es/luisherrero/apirest1/model/Aula.java
package es.luisherrero.apirest1.model;

import java.io.Serializable;
import java.util.List;

import jakarta.persistence.Id;
import jakarta.persistence.Table;

@jakarta.persistence.Entity
@Table(name="aulas")
public class Aula implements Serializable{
	private static final long serialVersionUID=1;
	@Id
	private int num;

	private String codigo;

	private String descripcion;

	private int planta;

	public Aula() {
	}

	public int getNum() {
		return this.num;
	}

	public void setNum(int num) {
		this.num = num;
	}

	public String getCodigo() {
		return this.codigo;
	}

	public void setCodigo(String codigo) {
		this.codigo = codigo;
	}

	public String getDescripcion() {
		return this.descripcion;
	}

	public void setDescripcion(String descripcion) {
		this.descripcion = descripcion;
	}

	public int getPlanta() {
		return this.planta;
	}

	public void setPlanta(int planta) {
		this.planta = planta;
	}

}

File: Creacion API/Python_e_imagenes/aula.py
class Aula:
    def __init__(self, num, codigo, descripcion, planta):
        self.num = num
        self.codigo = codigo
        self.descripcion = descripcion
        self.planta = planta

    def get_num(self):
        return self.num

    def set_num(self, value):
        self.num = value
        
    def get_codigo(self):
        return self.codigo

    def set_codigo(self, value):
        self.codigo = value
        
    def get_descripcion(self):
        return self.descripcion

    def set_descripcion(self, value):
        self.descripcion = value
        
    def get_planta(self):
        return self.planta

    def set_planta(self, value):
        self.planta = value